### ISSUE TITLE
Offline encryption activity

### ISSUE THREAD

**Issue by tinaverbo** [chadsansing/critical-web-literacy-curriculum#1]
# Critical Web Literacy Module | Offline Encryption Activity - DRAFT
## Overview

**Module Title:** Intro to Critical Web Literacy

**Activity Title:** Encrypt Decrypt

**Image:** [image web address or URL]

**Image credit:** [license + author name linked to the page where the image lives]

**Made by:** Tina Verbo & Mozilla

**Time:** [aim for 45-60 minutes]

**Activity X of Y** [this can be set later]

**Participate on the Web** 

**21st Century Skills:** [include all applicable skills]:
- [ ] Communication
- [ ] Problem-solving

**Web Literacy Skills:** [include all applicable skills; see the Web Literacy Map]
- [ ] Connect
- [ ] Protect

**Learning Objectives:** 
- Learn what is Encryption Basics
- Crack a message by encrypting and decrypting it
- Learn the benefits of encryption

**Audience:** 
Beginner – intermediate web user

**Materials:** [include both online and offline materials needed to complete your activity]
- https://drive.google.com/drive/folders/0B5anjmldUa-IaTY4b1F4R3FaWjQ?usp=sharing
- paper/sticky notes
- pen
- authentication code (for approach 2)
## Preparation

Print at least 2 copies of all the crack cards. Create random authentication code for each group. Only the respective encryptors and decryptors must know these. Depending on the number of learners, group them into 3-4 members. Make sure that someone will be the message sender, message encryptor, message decryptor and message receiver. 
## Introduction

Everyone’s online privacy depends on encryption. Learners will know more about how encryption works, why it's essential to a strong Web, and why it's worth protecting.

Learners will be simulate how basic encryption works by using crack codes and authentication codes. 
## Offline Activity

Make sure to group your participants with 4 members. Have a draw lots, be sure that participants will not tell anyone their roles and their groups. Respective crack cards will be given to both encryptors and decryptors. Only the message sender and receiver will know each other. 
##### Approach 1:
1.  All participants must not know the role of each other except for the message receivers and message senders. 
2.  Ask message senders to create a message with their crack number. After that, fold the paper and give them to the encryptors.
3.  Using their respective crack cards, encryptors must use a secret code to hide the message.
4.  Encrypted messages must be placed to the decryptors (note that crack number should be present on the cover of the folded paper). As a facilitator, make sure to create a challenge to you participants by getting a copy of one crack card and try to decrypt a message. 
5.  Decryptors must try to crack the message. Give the message to the receiver and they will reveal the message. 
##### Approach 2 (with authentication code):
1.  All participants must not know the role of each other except for the message receivers and message senders. 
2.  Ask message senders to create a message with their crack number. After that, fold the paper and give them to the encryptors.
3.  Using their respective crack cards, encryptors must use a secret code to hide the message, make sure the encrypted message must have the authentication code given to their group (only the encryptors and decryptors will know this) on the cover of the folded paper.
4.  Encrypted messages must be placed to the decryptors (note that crack number should be present). As a facilitator, make sure to create a challenge to you participants by getting a copy of one crack card and try to decrypt a message. 
5.  Decryptors must try to crack the message. Give the message to the receiver and they will reveal the message. 
## Reflection
- In your own words, can you summarize what you learned?
- What seemed easiest about today’s work? Most difficult? Why?
- What would you keep from today’s activity? What would you get rid of or change?
- How beneficial is encryption?

**Comment by chadsansing** [comment]
@tinaverbo, I love the secrecy and mystery of this activity. How did you decide on 4 players per group?

I have two questions:
- Is there a way to show the cost of not using encryption in this activity?
- How would you connect the lesson to the idea of "critical web literacy?" For example, is there an opportunity to show learners how to use critical thinking skills to analyze or evaluate whether or not a message or app is encrypted?

Great lesson!

**Comment by chadsansing** [comment]
@tinaverbo, I also saved a copy [here](https://github.com/chadsansing/critical-web-literacy-curriculum/blob/master/activities/offline-encryption.md) to get our activities folder started. Thank you!
